text/0005-global-init.md

```diff
@@ -85,9 +86,13 @@ not depend on the SDK being installed.
 
 ### Testing support
 
-Testing should be performed without depending on the global SDK.
-
-### Synchronization
+Testing should be performed without depending on the global SDK, if
+possible.  If it is necessary to install a global `Tracer` or `Meter`
+factory for code that explicitly relies on the global factories for
+testing, test libraries should provide a compatible SDK that can be
+registered once with support for dynamically adding and removing
+exporters.  The use of named `Tracer` and `Meter` instances will help
```

Comment:
How will the use of named tracers and meters help to avoid test interference? They are not encapsulated from each other, they just add additional information to Spans.

Reply:
Sorry, you're right. I removed this sentence.